api.js

```diff
@@ -8,13 +10,24 @@ var ecstatic = require('ecstatic')
 
 module.exports = Api
 
-function Api (osm, media) {
-  if (!(this instanceof Api)) return new Api(osm, media)
+function Api (osm, media, opts) {
+  if (!(this instanceof Api)) return new Api(osm, media, opts)
+  if (!opts) opts = {}
   this.osm = osm
   this.media = media
+  var id = opts.id || 'MapeoDesktop_' + randombytes(8).toString('hex')
+  var host = opts.host
+  this.sync = sync(osm, media, {id, host})
+  this.sync.listen()
 }
 
+
 // Observations
+
+Api.prototype.observationDelete = function (req, res, m) {
+  res.end('not implemented')
```

Comment:
@noffle have any time to implement this?

Reply:
I think we decided that it wasn't necessary for V1, but if it's easy to do it could be helpful just to complete CRUD? If it's not, then I'd leave it out for June 30th release date

Reply:
It's for MapFilter desktop